fontforgeexe/acorn2sfd.c

```diff
@@ -562,6 +562,8 @@ return;
 	    free(widths);
 	}
     }
+    if (m != 0)
+        free(mapping);
```

Comment:
Now I found how to edit directly a commit, no problem for this :-)
(great function!!)